src/components/Comments/WriteComment.js

```diff
@@ -72,11 +75,16 @@ const WriteComment = ({
       }
     });
   }, []);
+  React.useEffect(() => {
+    if (isKeyboardOpen) {
+      commentInputRef.current.focus();
+    }
+  }, [isKeyboardOpen, commentInputRef.current]);
   const saveToStorage = (valueData) => {
     AsyncStorage.setItem(storageKey, valueData);
   };
   return (
-    <View style={styles.columnContainer}>
+    <View style={isViewOnly ? styles.isViewOnlyColumnContainer : styles.columnContainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the container view is now conditionally set based on the `isViewOnly` prop. This is a good way to handle different styles based on the state of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->